File: 07. Perulangan Lanjut/Pola II/PolaII.java
import java.util.Scanner;

public class PolaII
{
    public static void main (String [] args)
    {
        Scanner scanner = new Scanner (System.in);

        short bilangan = scanner.nextShort ();

        scanner.close ();

        for (short i = 1; i <= bilangan; i++)
        {
            for (short j = 1; j <= bilangan - i; j++)
            {
                System.out.print (" ");
            }

            for (short j = 1; j <= i; j++)
            {
                System.out.print ("*");
            }

            System.out.println ();
        }
    }
}

File: 07. Perulangan Lanjut/Pola II/Pola-II.cpp
#include <iostream>

int main ()
{
    unsigned short bilangan = 0;

    std::cin >> bilangan;

    for (unsigned short i = 1; i <= bilangan; i++)
    {
        for (unsigned short j = 1; j <= bilangan - i; j++)
        {
            std::cout << " ";
        }

        for (unsigned short j = 1; j <= i; j++)
        {
            std::cout << "*";
        }
        
        std::cout << "\n";
    }

    return 0;
}